registry.json

```diff
@@ -48,6 +48,8 @@
   "backbone": "npm:backbone",
   "backbone.babysitter": "github:marionettejs/backbone.babysitter",
   "backbone.wreqr": "github:marionettejs/backbone.wreqr",
+  "big.js": "github:MikeMcl/big.js",
+  "bignumber.js": "github:MikeMcl/big.js",
```

Comment:
It is best to only call something by one name in the registry. Can we go with one of `big.js` or `bignumber.js`. Also do we really need the `.js` in the name here?